docroot/themes/contrib/civictheme/includes/views.inc

```diff
@@ -99,8 +99,9 @@ function _civictheme_filter_views_exposed_form_alter(&$form, FormStateInterface
   $listing_filters = NULL;
   if (!empty($view->civictheme_listing_entity)) {
     $listing_show_filters = civictheme_get_field_value($view->civictheme_listing_entity, 'field_c_p_listing_show_filters');
-    if ($listing_show_filters) {
-      $listing_filters = explode(', ', civictheme_get_field_value($view->civictheme_listing_entity, 'field_c_p_listing_filters_exp', FALSE, ''));
+    $exposed_listing_filters = civictheme_get_field_value($view->civictheme_listing_entity, 'field_c_p_listing_filters_exp');
```

Comment:
@joshua-salsadigital 
why do we need this change? I have implemented a fix to make sure that a value returned from the `civictheme_get_field_value()` resolves to an empty array

Reply:
if this really required - please add exception for PHPMD

Reply:
Not explicitly needed, Was not aware of the fix, Reverted the change.